db/migrate/20220503230849_add_reviewed_by_at_and_by_to_contributions.rb

```diff
@@ -0,0 +1,6 @@
+class AddReviewedByAtAndByToContributions < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :contributions, :reviewed_by, references: :users, index: true
```

Comment:
It's expected that the model itself reflects this relation by adding a `belongs_to`, `has_one` or `has_many` on it.

Reply:
Done :fist_oncoming: :tada: